meetings/2024-06-24.md

```diff
@@ -0,0 +1,21 @@
+# Solid Practitioners
+
+## Meetng Date
+* 2024-06-24
+
+## Facilitator 
+* Jeff Zucker - dubzed@gmail.com
+
+## Scribe
+* 
+
+## Participants (please add yourself)
+
+## Agenda
+
+* Announcements
+    * Call for Scribe
+
+* New Member Check-in (brief intro, current work)  
+
+* Discussing ideas around shape research & potential collaborations
```

Comment:
```suggestion
* Discussing ideas around shape research & potential collaborations
  * [SHACL](https://www.w3.org/TR/shacl/) (W3C REC) is insufficient or otherwise non-viable because ... ?
  * [ShEx](https://shex.io/shex-semantics/index.html) (W3C CG Report) is insufficient or otherwise non-viable because ... 
  * Have other options already been considered? Rejected?
```